styles.css

```diff
@@ -1,3 +1,793 @@
+    background-color: #141B22;
+    cursor: pointer;
+}
+
+.connect-button {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    gap: 8px;
+    font-size: 16px;
+    color: #000;
+    font-family: "Bakbak One", Arial, sans-serif;
+    font-weight: 400;
+    line-height: 24px;
+    padding: 12px 24px;
+    border: none;
+    height: 48px;
+    width: 149px;
+    background: #21E786;
+    cursor: pointer;
+}
+
+.breadcrumbs-nav a {
+    font-size: 14px;
+    color: #888B8E;
+    font-family: "Bakbak One", Arial, sans-serif;
+    font-weight: 400;
+    line-height: 20px;
+}
+
+.main-heading,
+.heading {
+    font-family: "Bakbak One", Arial, sans-serif;
+    font-weight: 400;
+    font-size: 44px;
+    line-height: 56px;
+    color: #fff;
+    margin: 0;
+}
+
+.sub-heading,
+.how-to-text {
+    font-family: "Bakbak One", Arial, sans-serif;
+    font-size: 24px;
+    font-weight: 400;
+    line-height: 32px;
+    color: #fff;
+}
+
+.how-to-heading {
+    font-family: "Bakbak One", Arial, sans-serif;
+    font-size: 22px;
+    font-weight: 400;
+    line-height: 30px;
+    text-align: center;
+    color: #21E786;
+
+}
+
+.ot-member-pos,
+.faq-description,
+.joc-text,
+.sub-text {
+    font-family: Poppins, Arial, sans-serif;
+    font-weight: 400;
+    font-size: 18px;
+    line-height: 28px;
+    color: #C2C3C5;
+}
+
+.copyright-text {
+    font-family: Poppins, Arial, sans-serif;
+    font-size: 14px;
+    font-weight: 400;
+    line-height: 22px;
+
+}
+
+.visually-hidden {
+    visibility: hidden;
+}
+
+.nft-card,
+.about-us-list-content,
+.step-list,
+.roadmap-phase-list,
+.roadmap-nav-list,
+.faqs-list,
+.social-links-logo,
+.social-links,
+.filter-list,
+.catalog-list,
+.blog-list,
+.tag-list,
+.social-list,
+.categories-list,
+.recent-post-list {
+    list-style: none;
+}
+
+.list-term {
+    font-family: "Bakbak One", Arial, sans-serif;
+    font-size: 20px;
+    font-weight: 400;
+    line-height: 28px;
+    color: #C2C3C5;
+}
+
+.list-description {
+    font-family: "Bakbak One", Arial, sans-serif;
+    font-size: 56px;
+    font-weight: 400;
+    line-height: 68px;
+    color: #fff;
+    text-shadow: 0 4px 16px #FFFFFF66;
+}
+
+.list-item-number {
+    font-family: "Bakbak One", Arial, sans-serif;
+    font-weight: 400;
+    font-size: 80px;
+    line-height: 96px;
+    color: #21E786;
+    opacity: 30%;
+}
+
+.list-item-sub {
+    font-family: Poppins, Arial, sans-serif;
+    font-weight: 400;
+    font-size: 18px;
+    line-height: 28px;
+    text-align: center;
+    color: #C2C3C5;
+}
+
+.joc-text-diff,
+.sub-info {
+    font-family: Poppins, Arial, sans-serif;
+    font-size: 18px;
+    font-weight: 400;
+    line-height: 28px;
+    text-align: center;
+    color: #F7F7F9;
+}
+
+.page-footer-text {
+    font-family: Poppins, Arial, sans-serif;
+    font-size: 16px;
+    font-weight: 400;
+    line-height: 26px;
+}
+
+.list-heading,
+.main-content-heading,
+.rating-by,
+.caption-name,
+.caption-name {
```

Comment:
Long lists of selectors can be a sign that you're applying the same styles to many different elements. If these elements always share all the same styles, it’s better to give them a common class. If only some properties are shared, consider splitting the styles to avoid specificity conflicts later on. This will make your CSS more modular and easier to debug.